crates/agentgateway/src/proxy/httpproxy.rs

```diff
@@ -497,10 +497,47 @@ impl HTTPProxy {
 		// Record request now. We may do it later as well, after we have more expressions registered.
 		Self::apply_request_to_cel(log, &mut req).await;
 
+		// Check for per-frontend sampling overrides first, before we decide sampling
+		let frontend_policies = inputs
+			.stores
+			.read_binds()
+			.frontend_policies(self.inputs.cfg.gateway());
+		if let Some(tp) = frontend_policies.tracing.as_ref() {
+			// Apply sampling overrides if present
+			if let Some(rs) = &tp.config.random_sampling {
+				log.cel.tracing_sampler.random_sampling = Some(rs.clone());
```

Comment:
can be a followup but I am leaning towards deleting `config.tracing` in favor of only `frontendPolicies.tracing`. With https://github.com/agentgateway/agentgateway/pull/746 this should be obvious on upgrade.

We can do the same with logging, though logging is slightly awkward since we will want to keep `level` and `format` but I think thats fine (its config.logging for all logging, frontendPolicies.accessLog for access logs)